doc/lsst.source.injection/reference/41_match_injected_outputs.rst

```diff
@@ -0,0 +1,201 @@
+This saves memory and runtime by skipping the expensive ``makeWarp`` and ``assembleCoadd`` tasks which usually occur before ``inject_coadd`` in the default ``step3``.
+
+.. code::
+
+    pipetask --long-log --log-file $LOGFILE \
+    run --register-dataset-types \
+    -b $REPO \
+    -i $INPUT_DATA_COLL,$INJECTION_CATALOG_COLL \
+    -o $PROCESSED_DATA_COLL \
+    -p $DRP_PIPE_DIR/pipelines/HSC/DRP-RC2_subset+injected_deepCoadd.yaml#injected_step3 \
+    -d "instrument='HSC' AND skymap='hsc_rings_v1' AND tract=9813 AND patch=42 AND band='i'"
+
+*where*
+
+    `$LOGFILE`
+        The full path to a user-defined output log file.
+
+    `$REPO`
+        The path to the butler repository.
+
+    `$INPUT_DATA_COLL`
+        The name of the input data collection.
+
+    `$INJECTION_CATALOG_COLL`
+        The name of the input injection catalog collection.
+
+    `$PROCESSED_DATA_COLL`
+        The name of the injected output collection.
+
+.. note::
+    While this example makes use of ``injected_step3`` within a coadd-level injection pipeline, it is also possible to run ``injected_step1`` within an exposure- or visit-level pipeline.
+    Regardless of injection type, you can then elect to continue processing by running any following ``injected_`` steps.
+
+-------------------------------
+ Consolidate injected catalogs
+-------------------------------
+Even if a single catalog is injected, the butler splits catalogs which cover multiple photometry bands, or which cover large areas of sky for memory efficiency.
+For example, if a coadd-level injection catalog covers a whole tract across multiple photoemtry bands, the injected catalogs will be split and stored with the dimensions ``{patch, band}``.
+Before matching the injected input catalogs to the processed output catalog, the per-patch and per-band inputs must be consolidated into a single tract level catalog.
+This can be done by using ``pipetask run`` to run ``ConsolidateInjectedCatalogsTask`` from ``pipelines/consolidate_injected_deepCoadd_catalogs.yaml``.
+
+.. code::
+
+    pipetask --long-log --log-file $LOGFILE \
+    run --register-dataset-types \
+    -b $REPO \
+    -i $PROCESSED_DATA_COLL \
+    -o $CONSOLIDATED_CATALOG_COLL \
+    -p $SOURCE_INJECTION_DIR/pipelines/consolidate_injected_deepCoadd_catalogs.yaml \
+    -d "instrument='HSC' AND skymap='hsc_rings_v1' AND tract=9813 AND patch=42 AND band='i'"
+
+*where*
+
+    `$LOGFILE`
+        The full path to a user-defined output log file.
+
+    `$REPO`
+        The path to the butler repository.
+
+    `$PROCESSED_DATA_COLL`
+        The name of the input injected catalog collection.
+
+    `$CONSOLIDATED_CATALOG_COLL`
+        The name of the consolidated injected output collection.
+
+----------
+ Matching
+----------
+Matching two tract-level catalogs can be done trivially with a ``pipetask run`` command as below:
```

Comment:
I'd recommend an intro sentence, that starts along the lines of:

"Now that we have our consolidated tract-level injected catalog and a reference tract-level standard catalog, we can move on to matching these two sets of catalogs together."